.eslintrc.json

```diff
@@ -11,20 +11,19 @@
   },
   "ignorePatterns": [
     "!.storybook",
-    "packages/bvaughn-architecture-demo/playwright/protocol-fixtures/"
+    "packages/bvaughn-architecture-demo/playwright/protocol-fixtures/",
+    "src/devtools/client/debugger/src/utils/editor/index.d.ts"
   ],
   "parserOptions": {
     "ecmaFeatures": {
       "jsx": true
     },
     "ecmaVersion": 2018,
-    "project": ["tsconfig.json"],
+    "project": ["tsconfig.eslint.json"],
     "sourceType": "module"
   },
   "plugins": ["import", "react", "prettier", "@typescript-eslint"],
   "rules": {
-    "@typescript-eslint/no-base-to-string": "warn",
```

Comment:
Sorry I think there's some confusion. Can we leave `@typescript-eslint/no-base-to-string` enabled? Is there a reason you think we shouldn't?

Reply:
Oh, shit, sorry, I deleted the lines that were disabling it, but I forgot that actually I needed to change them from `off` back to `warn` explicitly.